### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#303] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152321692-af3d89d8-efc0-4887-931f-9915d431cd63.png)

**Comment by Exyss** [comment]
![Esercizio 3](https://user-images.githubusercontent.com/65043207/152644030-1cfb21f9-9719-4cb1-8abf-de15cf522d9c.jpg)

**Comment by Duvan-Zapat** [comment]
> ![Esercizio 3](https://user-images.githubusercontent.com/65043207/152644030-1cfb21f9-9719-4cb1-8abf-de15cf522d9c.jpg)

Potresti far vedere i calcoli?
